R/zchunk_LA161.Cstorage_china.R

```diff
@@ -0,0 +1,113 @@
+#' module_gcam.china_LA161.Cstorage
+#'
+#' Calculates onshore CO2 storage by grid region.
+#'
+#' @param command API command to execute
+#' @param ... other optional parameters, depending on command
+#' @return Depends on \code{command}: either a vector of required inputs,
+#' a vector of output names, or (if \code{command} is "MAKE") all
+#' the generated outputs: \code{L161.Cstorage_province}. The corresponding file in the
+#' original data system was \code{LA161.Cstorage.R} (gcam-china level1).
+#' @details Calculates onshore CO2 storage by grid region.
+#' @importFrom assertthat assert_that
+#' @importFrom dplyr filter mutate select
+#' @importFrom tidyr gather spread
+#' @author Liu August 2018
+
+module_gcam.china_LA161.Cstorage <- function(command, ...) {
+  if(command == driver.DECLARE_INPUTS) {
+    return(c(FILE = "gcam-china/province_names_mappings",
+             FILE = "gcam-china/CCS_provincial_data",
+             FILE = "gcam-china/CCS_provincial_data_add",
+             FILE = "gcam-china/unallocated_CStorage_province"))
+  } else if(command == driver.DECLARE_OUTPUTS) {
+    return(c("L161.Cstorage_province"))
+  } else if(command == driver.MAKE) {
+
+    # Silence package checks
+    province <- province.name <- grid_region <- Cost_2005USDtCO2 <- CO2_Mt <- MtC <- MtCO2 <- Cumul_MtC <- quantile <-
+      grade <- Cost_1990USDtC <- add_storage <- NULL
+
+    all_data <- list(...)[[1]]
+
+    # -----------------------------------------------------------------------------
+    # 1.Load required inputsMtCO2
+    province_names_mappings <- get_data(all_data, "gcam-china/province_names_mappings")
+    Dooley_CCS_China        <- get_data(all_data, "gcam-china/CCS_provincial_data")
+    CCS_China_add           <- get_data(all_data, "gcam-china/CCS_provincial_data_add")
```

Comment:
Change "CCS_China_add" to "CCS_China_province_add"

Reply:
get